### ISSUE TITLE
Accessibility: There is no focus state for menu items, making it difficult for keyboard users

### ISSUE THREAD

**Issue by cailborg** [alexplumb/material-ui-phone-number#40]
List items have a tab index of "-1" which seems to make them not keyboard focusable
